x/ibc-rate-limit/contracts/rate-limiter/src/management.rs

```diff
@@ -88,6 +90,7 @@ pub fn try_reset_channel_quota(
                 channel_id: channel_id.clone(),
             }),
             Some(mut flows) => {
+                // Q: What happens here if quote_id not found? seems like we return ok?
```

Comment:
We shouldn't end up there if the quota is not found (that would be caught by the `None` arm above), but what could happen is that the vector is empty.

In that case, yeah, we return `Ok` with no changes applied. Update takes this and updates the stored data (which is the same as before). A bit unnecessary to update it here, but it allows is to return the error directly from that function (see the `?` at the end) instead of having to catch and handle it. Storing an empty vec should be a very cheap operation anyway